unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -712,6 +715,9 @@ public bool ProcessMessage(MessagingBus.QueuedSceneMessage_Scene msgObject, out
                     case MessagingTypes.OPEN_EXTERNAL_URL:
                         OnOpenExternalUrlRequest?.Invoke(scene, payload.OpenExternalUrl.Url);
                         break;
+                    case MessagingTypes.OPEN_NFT_DIALOG:
```

Comment:
I see we don't have this 2 cases in our WSSController.cs to have this working from the unity editor, we should add these, right?

Reply:
OK it's not needed because it uses protobuff messaging